src/app/(routes)/page.tsx

```
@@ -1,17 +1,121 @@
+): 'common' | 'rare' | 'superRare' | 'legendary' => {
+  switch (
+    grade.toLowerCase() // 소문자로 변환
+  ) {
+    case 'common':
+      return 'common';
+    case 'rare':
+      return 'rare';
+    case 'superrare':
+    case 'super_rare':
+    case 'SUPER_RARE':
+      return 'superRare';
```

Biome (1.9.4)

[error] This expression always returns false, because the string is converted and will never match

This call convert the string to lower case

... but this value is not in lower case

Unsafe fix: Use lower case string value.


(lint/correctness/noStringCaseMismatch)